.travis.yml

```diff
@@ -280,7 +280,10 @@ script:
   - ./ci/suppress_output bash src/ray/test/run_object_manager_tests.sh
 
   # cc bazel tests (w/o RLlib)
-  - ./ci/suppress_output bazel test --config=ci --build_tests_only -- //:all -rllib/...
+  # Use separated test tag for stats to avoid influences of static varibles in
+  # metric reporter.
+  - ./ci/suppress_output bazel test --config=ci --build_tests_only --test_tag_filters=-stats -- //:all -rllib/...
+  - ./ci/suppress_output bazel test --config=ci --build_tests_only --test_tag_filters=stats --jobs=1 -- //:all -rllib/...
```

Comment:
this change is no longer needed?

Reply:
It should be. But there are some side-effects if stats handler threads have never been halted.